bakerydemo/base/models.py

```diff
@@ -23,48 +23,16 @@
 from .blocks import BaseStreamBlock
 
 
-class BasePageFieldsMixin(models.Model):
-    """
-    An abstract base class for common fields
-    """
-    introduction = models.TextField(
-        help_text='Text to describe the page',
-        blank=True)
-    image = models.ForeignKey(
-        'wagtailimages.Image',
-        null=True,
-        blank=True,
-        on_delete=models.SET_NULL,
-        related_name='+',
-        help_text='Landscape mode only; horizontal width between 1000px and '
-        '3000px.'
-    )
-    # The StreamField is defined within base/blocks.py
-    body = StreamField(
-        BaseStreamBlock(), verbose_name="Page body", blank=True
-    )
-    # The content_panels makes the fields accessible to the editor within the
-    # admin area. If you don't include a field here it won't be displayed
-    content_panels = Page.content_panels + [
-        FieldPanel('introduction', classname="full"),
-        ImageChooserPanel('image'),
-        StreamFieldPanel('body'),
-    ]
-
-    class Meta:
-        abstract = True
-
-
 @register_snippet
 class People(ClusterableModel):
     """
     A Django model to store People objects.
     It uses the `@register_snippet` decorator to allow it to be accessible
-    as via the Snippets UI (e.g. /admin/snippets/base/people/)
+    via the Snippets UI (e.g. /admin/snippets/base/people/)
 
     `People` uses the `ClusterableModel`, which allows the relationship to another
     model (e.g. a PageModel) to be stored independently of the database. This
-    allows us to preview foreignKey relationships without first saving the parent
+    allows us to preview ForeignKey relationships without first saving the parent
```

Comment:
How about "allows us to *create* ForeignKey relationships without first saving the parent" ? Or maybe I'm not sure what's meant by "preview."